src/lib/swapper/swappers/CowSwapper/types.ts

```diff
@@ -27,9 +27,11 @@ export enum CowNetwork {
   Xdai = 'xdai',
 }
 
-export type CowChainId = KnownChainIds.EthereumMainnet | KnownChainIds.GnosisMainnet
+export const cowChainIds = [KnownChainIds.EthereumMainnet, KnownChainIds.GnosisMainnet] as const
```

Comment:
Prefer upper snake case for consts but meh

Reply:
all of the other ones are the same casing so more following pattern than anything. would rather update all if we want at the same time, but not for this PR with everything else